ice-adapter/src/main/java/com/faforever/iceadapter/CliLauncher.java

```diff
@@ -0,0 +1,37 @@
+package com.faforever.iceadapter;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import picocli.CommandLine;
+
+import java.util.concurrent.Callable;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class CliLauncher implements IceOptions, Callable<Integer> {
```

Comment:
This needs an `@Command` annotation to work. I would also suggest adding the mixinStandardHelpOptions=true as well as a version provider